states/WI/governor.yaml

```diff
@@ -0,0 +1,32 @@
+contact_form:
+  steps:
+    - visit: "https://appengine.egov.com/apps/wi/Governor/contact"
+    - fill_in:
+        - name: "firstname"
```

Comment:
Let's make `name`s match the real `name` attributes of the form elements.

Reply:
Any reason why?  I thought it was nice that it was printing human useful strings for this, as opposed to the UUID it gives otherwise (for this particular form).  I don't have a strong preference otherwise.

Reply:
That's what the https://github.com/unitedstates/contact-congress/blob/master/documentation/schema.md#fill_in schema says. We could decide to change that -- in that case we should delete all the `name`s from our configs.

Reply:
aha, then yes, your guidance is what I'll follow.